launch-marathon-app.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-curl -X POST -d @$1 http://127.0.0.1:8080/v2/apps -H 'Content-Type: application/json'
+curl -X POST -d @$1 http://127.0.0.1:8080/v2/apps -H 'Content-Type: application/json' | jq .
```

Comment:
Not everyone has `jq` installed on their machine, we install it in the Vagrant box but not on the host, which is where this  is run.